dietpi/dietpi-software

```diff
@@ -12096,6 +12309,15 @@ _EOF_
 
 			AGP chromium
 
+		elif (( $1 == 157)); then
+
+			rm -R /srv/homeassistant
+			userdel -r -f homeassistant
+
+			# How is usage of these items tracked across software packages, what if another package installs
+			# a item in this list and uninstalling HA kills it with this removal?
+			AGP python3 python3-pip libssl-dev cmake libc-ares-dev uuid-dev daemon curl libgnutls28-dev libgnutlsxx28 nmap net-tools sudo libglib2.0-dev cython3 libudev-dev python3-sphinx python3-setuptools libmysqlclient-dev swig libssl-dev python-dev libusb-1.0-0 gcc python3-dev libssl-dev libffi-dev
```

Comment:
ok, thanks for the comment. Ill remove this in the next iteration. Most of the actual install happens in python.